### ISSUE TITLE
Adding kubefwd extension results in k8s context error

### ISSUE THREAD

**Issue by alisaifee** [tilt-dev/tilt-extensions#253]
Upon adding the kubefwd extension using the instructions [here](https://github.com/tilt-dev/tilt-extensions/tree/master/kubefwd) I got an error in the `kubefwd:config` local resource: 

```
Error: Refusing to run 'local' because dev-3 might be a production kube context.
If you're sure you want to continue add:
	allow_k8s_contexts('dev-3')
before this function call in your Tiltfile. Otherwise, switch k8s contexts and restart Tilt.
```

I had however, already added `dev-3` to the list of allowed contexts using `allow_k8s_contexts`.

When I modified the extension's Tiltfile directly by adding an `allow_k8s_contexts` call directly before the `local` command [here](https://github.com/tilt-dev/tilt-extensions/blob/master/kubefwd/Tiltfile#L8), however - the extension loaded up and worked just fine.

**Comment by nicks** [comment]
thanks for the report! hopefully this should be easy to fix.

**Comment by nicks** [comment]
we just released v0.22.10, which should fix this problem

**Comment by alisaifee** [comment]
Thank you!
